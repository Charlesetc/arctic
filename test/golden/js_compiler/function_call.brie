define main [
  define f : x y z [ x ]
  define y [ 2 ]
  f 2 "hi" y
]
