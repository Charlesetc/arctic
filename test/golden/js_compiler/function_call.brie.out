const global_functions = {1: {},2: {"Integer,String,Function": fn_2_Integer_String_Function},3: {}};
function fn_2_Integer_String_Function(x,y,z) {
	return x;
}
function main() {
	var f = new_closure(2);
	var y = new_closure(3);
	return f.partial(2, "Integer").partial("hi", "String").partial(y, "Function").call();
}
console.log(main());